libs/@blockprotocol/type-system/src/native/entity-type.ts

```diff
@@ -0,0 +1,48 @@
+import { EntityType, VersionedUri } from "../../wasm/type-system";
+
+/**
+ * Returns all the IDs of all types referenced in a given property type.
+ *
+ * @param {EntityType} entityType
+ */
+export const getReferencedIdsFromEntityType = (
```

Comment:
I would have liked to just expose the `Rust` implementations of these functions, but we're trying to avoid requiring all importers of the `graph` package from having to initialise the type system at the moment..

Perhaps it's fine to require initialisation of the type system for our internal API methods?

cc. @CiaranMn for thoughts, unsure if this will complicate it for environments like WP?

Reply:
WP is going to have to use `buildSubgraph` in the browser. It won't need ontology types, but I think the addition of anything WASM-dependent here would necessitate shipping the WASM to the browser even if it isn't used? So you might not have a crash, but you're adding a lot to the bundle size. If that's true I'd avoid it.

Reply:
Correct. I still think we can get the bundle size down quite a lot, and can benefit from caching of the WASM module, but without that investment this seems like the correct compromise for now then